package.json

```diff
@@ -8,29 +8,28 @@
     "@types/jest": "24.0.13",
     "@types/node": "12.0.8",
     "@types/react": "16.8.19",
-    "@types/react-addons-css-transition-group": "^15.0.5",
     "@types/react-dom": "16.8.4",
     "@types/react-router-dom": "^5.1.6",
-    "@types/socket.io-client": "^1.4.34",
+    "@types/socket.io-client": "^3.0.0",
     "@types/styled-components": "^5.1.5",
     "chart.js": "^2.9.4",
     "enzyme-adapter-react-16": "^1.15.5",
     "react": "^16.14.0",
-    "react-addons-css-transition-group": "^15.6.2",
     "react-chartjs-2": "^2.11.1",
     "react-dom": "^16.14.0",
-    "react-google-charts": "^3.0.15",
+    "react-google-charts": "^5.1.0",
     "react-redux": "^7.2.2",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.0.1",
+    "react-transition-group": "^4.4.5",
     "redux": "^4.0.5",
-    "socket.io-client": "^2.3.1",
+    "socket.io-client": "^3.1.3",
     "styled-components": "^4.4.1",
-    "typescript": "^3.9.7"
+    "typescript": "4.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "CI= react-scripts build",
+    "start": "export NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
```

Comment:
When I run this locally, it gives me an error. Have you tried running this locally?

Reply:
It doesn’t work for me either. I’m fairly certain this is a work in progress PR and we will discuss it tomorrow in the call. I’m wondering if we should try to get things up and running with the old version of Node that would have been used for this and then worry about upgrading afterwards. I’m not sure which is the fastest / most efficient approach here considering our limited time.